js/scripts.js

```diff
@@ -0,0 +1,16 @@
+// Excercise 8.6
+// isNaN method added 
+
+var a = prompt("podaj liczbę");
+var b = prompt("podaj liczbę");
+var number = (a * a) + (2 * a * b) - (b * b);
+console.log(number);
+if (isNaN('a') && isNaN('b') && number > 0) {
```

Comment:
isNaN returns true when first arguments is NaN - you write it wrong :) in your code everything goes to last conditional